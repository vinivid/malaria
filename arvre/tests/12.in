d Walter
i Carlos
i Iara
d Andre
d Nadia
i Helio
d Guta
i Beto
d Andre
d Daniela
i Omar
i Helio
i Xena
d Kelle
i Jeremias
i Tadeu
d Jeremias
i Beto
i Sara
i Helio
d Daniela
i Tadeu
i Eliane
i Omar
d Xena
d Iara
i Walter
d Sara
d Fred
i Beto
i Sara
d Vania
d Queila

