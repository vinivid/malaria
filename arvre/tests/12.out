digraph {
"0 Carlos" ->"1 Beto" 
"0 Carlos" ->"2 Helio" 
"2 Helio" ->"5 Eliane" 
"2 Helio" ->"6 Omar" 
"6 Omar" ->"14 Tadeu" 
"14 Tadeu" ->"29 Sara" 
"14 Tadeu" ->"30 Walter" 
}

digraph {
"0 Helio" ->"1 Carlos" 
"0 Helio" ->"2 Tadeu" 
"1 Carlos" ->"3 Beto" 
"1 Carlos" ->"4 Eliane" 
"2 Tadeu" ->"5 Omar" 
"2 Tadeu" ->"6 Walter" 
"5 Omar" ->"12 Sara" 
}

digraph {
"0 Helio" ->"1 Carlos" 
"0 Helio" ->"2 Tadeu" 
"1 Carlos" ->"3 Beto" 
"1 Carlos" ->"4 Eliane" 
"2 Tadeu" ->"5 Omar" 
"2 Tadeu" ->"6 Walter" 
"5 Omar" ->"12 Sara" 
}

